shuttering/prod/apply-divorce-service-gov-uk.yml

```diff
@@ -19,18 +19,18 @@ A: []
 
 cname:
   - name: "*"
-    shutter: false
+    shutter: true
```

Comment:
Can we change this back to false, "*" records don't currently work with the shuttering process. The service will still be shuttered by leaving the rest of the records as true

Reply:
done 👍